Cargo.lock

```diff
@@ -0,0 +1,3799 @@
+# This file is automatically @generated by Cargo.
```

Comment:
We should not commit the `Cargo.lock` in case of libraries I would say.

Reply:
Alright, I will add it to the gitignore.